### PR TITLE
add ci check in check_api_approvals.sh, test=develop

### PR DESCRIPTION
**Pull request by juncaipeng**
add ci check in check_api_approvals.sh, test=develop

### CODE REVIEW

File: tools/check_api_approvals.sh [hunk 1]
```diff
@@ -112,6 +114,9 @@ for API_FILE in ${API_FILES[*]}; do
       elif [ "${API_FILE}" == "python/paddle/fluid/tests/unittests/white_list/op_check_grad_white_list.py" ];then
           echo_line="You must have one RD (zhangting2020 (Recommend), luotao1 or phlrain) approval for the python/paddle/fluid/tests/unittests/white_list/op_check_grad_white_list.py, which manages the white list of operators without gradient. For more information, please refer to: https://github.com/PaddlePaddle/Paddle/wiki/Gradient-Check-Is-Required-for-Op-Test. \n"
           check_approval 1 26615455 6836917 43953930
```
Comment (luotao1) [inline tools/check_api_approvals.sh:116]: https://github.com/PaddlePaddle/Paddle/blob/4eba861b53db5122651f760aeaafa983ccfaf508/tools/check_api_approvals.sh#L92 补充上juncaipeng的githubid号

**Review by luotao1 — COMMENTED** [review]
请加上`test=document_fix`来加速CI

File: tools/check_api_approvals.sh (hunk 1 above)
Comment (juncaipeng) [reply tools/check_api_approvals.sh:116]: > https://github.com/PaddlePaddle/Paddle/blob/4eba861b53db5122651f760aeaafa983ccfaf508/tools/check_api_approvals.sh#L92
> 
> 补充上juncaipeng的githubid号

Done

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**